### PR TITLE
Manage connection timeout

### PR DESCRIPTION
**Pull request by Edmilson-JR**
PR as discussed at issue #676

### CODE REVIEW

**Comment by Edmilson-JR** [comment]
@ekohl there was an intruder wrong comma

Already fixed it

**Comment by mmoll** [comment]
@Edmilson-JR could you also remove `configtimeout`?

File: manifests/init.pp [hunk 1]
```diff
@@ -113,6 +113,18 @@
 #
 # $additional_settings::                    A hash of additional main settings.
 #
+# $http_connect_timeout::                   The maximum amount of time to wait (agent)
+#                                           when establishing an HTTP connection.
+#                                           Replaces 'configtimeout' along with
+#                                           'http_read_timeout' (Puppet 4+).
```
Comment (ekohl) [inline manifests/init.pp]: The module is Puppet 5 only as of master so you can leave this out.

**Comment by ekohl** [comment]
Replaced by https://github.com/theforeman/puppet-puppet/pull/693

**Comment by Edmilson-JR** [comment]
@ekohl sorry for the absence.

The last days was very complicated here.

Saw that #693 was merged.

Thank you.

**Comment by ekohl** [comment]
We've all been there. Thanks for the initial PR.

**Pull request closed without merging**